pkg/analytics/googleanalytics.go

```diff
@@ -0,0 +1,38 @@
+package analytics
```

Comment:
Rename this file:
From `pkg/analytics/googleanalytics.go` to `pkg/analytics/analytics.go`